paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -50,8 +50,8 @@ def add_subparser(subparsers):
         required=True,
     )
     list_parser.add_argument(
-        '-l', '--clusterinstance',
-        help='Mark the service ready for deployment in this clusterinstance (e.g. '
+        '-l', '--deploy_group',
```

Comment:
You are going to have to make this backwards compatible. All of our existing jenkins jobs are doing --clusterinstance right now